packages/react-server/src/index.ts

```diff
@@ -1,3 +1,4 @@
+// release comment
```

Comment:
Assuming Lerna looks at commit hashes and not file diffs (?), you could do a `git commit --alow-empty -m "Release commit"` to tickle git with no code changes.

Reply:
It looks for file changes.